frame/staking/src/voter_bags.rs

```diff
@@ -1021,23 +1034,131 @@ mod voter_list {
 	}
 
 	#[test]
-	fn storage_is_cleaned_up_as_voters_are_removed() {}
+	fn storage_is_cleaned_up_as_voters_are_removed() {
+		todo!()
+	}
 
 	#[test]
 	fn insert_works() {
-		todo!()
+		ExtBuilder::default().build_and_execute(|| {
+			// given
+			assert_eq!(
+				VoterList::<Test>::iter().map(|n| n.voter().id).collect::<Vec<_>>(),
+				vec![11, 21, 101, 31]
+			);
+			// TODO maybe checking the actual bags here is overkill since this is aimed at VoterList
+			// api and not bags? (same Q for other tests here)
+			assert_eq!(get_bags(), vec![(10, vec![31]), (1_000, vec![11, 21, 101])]);
```

Comment:
I'm fine with testing as many stuff as possible. 

To help make it more tidy, maybe make a `fn iter_voter_list() -> Vec<AccountId>`.

Reply:
Also as per the intention of this test module: 

```
// This is the highest level of abstraction provided by this module. More generic tests are here,
// among those related to `VoterList` struct.
#[cfg(test)]
mod voter_list { ... }
```

I think we should manually check all storage items. This is the same API that the top level pallet will use (e.g. `VoterList::insert`), it is important to see exactly what is in storage before and after an insert.